### PR TITLE
Creates Initial SQLModels

### PR DESCRIPTION
**Pull request by imranrhem**
Creates SQLModels for tables, see comments for relationships.

### CODE REVIEW

File: app/models/fantasy_team.py
Comment (imranrhem) [inline app/models/fantasy_team.py:1]: Fantasy Team -> User
- Two-way many:one relationship
- Should be able to access a single user and it's info from the Fantasy Team 

Fantasy Team -> League
-Two-way one:many relationship
- Should be able to access a league from a team, and a league's fantasy teams

Fantasy Team -> RosterSlot
-One-way many:one relationship
-Can access many roster slot's info from the team but not team info from the roster slot (except that it belongs to a particular team)

File: app/models/league.py
Comment (imranrhem) [inline app/models/league.py:1]: League -> Fantasy Team
- Two-way one:many relationship
- Should be able to access a league from a team, and a league's fantasy teams

League -> RosterSettings
- One-way one:one relationship
- Can access a league's roster settings, but roster setting is only aware of what team it belongs to

League -> ScoringSettings
- One-way one:one relationship
- Can access a league's scoring settings, but roster setting is only aware of what team it belongs to

File: app/models/roster_slot.py
Comment (imranrhem) [inline app/models/roster_slot.py:1]: Roster slot is an interesting one. Open question: should a roster slot be able to access a player object? Thinking about when a game isn't running and there needs to be info when a player is selected. Does it just have to link an ID to the player table to join and pull info? Or does the player object need to be connected. For now:


Roster Slot -> NFLPlayerGame
- One-way one:one relationship
- Can access NFL player game info from roster slot, but nflplayergame has no idea roster slot exists 

Roster Slot -> Position
- Not really a way position - more that the position_id acts as a validator when a player is selected i.e) `roster_slot.position_id = player.position_id`

File: app/models/defensive_player_stats.py [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 class DefensivePlayerStats(SQLModel, table=True):
     __tablename__ = "defensive_player_stats"
-    game_id: int = Field(foreign_key="nfl_player_game.id")
+    game_id: int = Field(foreign_key="nfl_player_game.id", primary_key=True)
```
Comment (alexjnelson) [inline app/models/defensive_player_stats.py:5]: rename to player_game_id

**Pull request closed without merging**